Shubham-kumar/daily-learning-material/filter.js

```diff
@@ -0,0 +1,25 @@
+function filter(x,y)
+    {   
+
+        for (i =0; i<y.length; i++)
+        {
+          
+            delete x[y[0]]
```

Comment:
This will make us lose the information. We don't want that.

Reply:
I will update it soon.